docs/content/preview/yugabyte-platform/manage-deployments/manage-runtime-config.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Manage Runtime configuration
+headerTitle: Manage Runtime configuration
+linkTitle: Manage Runtime configuration
+description: Manage your universes by setting runtime configuration keys based on different scopes.
+menu:
+  preview_yugabyte-platform:
+    identifier: manage-runtime-config
+    parent: manage-deployments
+    weight: 100
+type: docs
+---
+
+YugabyteDB Anywhere has a configuration mechanism that is fully compatible with the existing [typesafe config](https://github.com/lightbend/config) format and has the following additional properties:
```

Comment:
"configuration mechanism" - what is this about?

Reply:
I believe this should written as `runtime configuration mechanism`. 
Cc @sb-yb